security/keycloak-multitenant/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftTest.java

```diff
@@ -0,0 +1,132 @@
+package io.quarkus.ts.openshift.security.keycloak;
+
+import com.gargoylesoftware.htmlunit.SilentCssErrorHandler;
+import com.gargoylesoftware.htmlunit.TextPage;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.WebRequest;
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
```

Comment:
Headless browser seems kinda heavy, is it required for this test?

Reply:
Yes, It's required for the webapp and JWT tenants because when accessing a protected resource, the app redirects to Keycloak for authentication. We can't verify this scenario using RestAssured.